packages/react-components/react-nav-preview/src/components/NavGroup/useNavGroup.styles.ts

```diff
@@ -0,0 +1,245 @@
+      content: '""',
+      position: 'absolute',
+    },
+    '@media (forced-colors: active)': {
+      ':hover::before': {
+        backgroundColor: 'Highlight',
+      },
+      ':active::before': {
+        backgroundColor: 'Highlight',
+      },
+    },
+  },
+});
+
+const useActiveIndicatorStyles = makeStyles({
+  base: {
+    '::after': {
+      backgroundColor: tokens.colorTransparentStroke,
+      ...shorthands.borderRadius(tokens.borderRadiusCircular),
+      content: '""',
+      position: 'absolute',
+      zIndex: 1,
+    },
+  },
+  selected: {
+    '::after': {
+      backgroundColor: tokens.colorCompoundBrandStroke,
+    },
+    ':hover::after': {
+      backgroundColor: tokens.colorCompoundBrandStrokeHover,
+    },
+    ':active::after': {
+      backgroundColor: tokens.colorCompoundBrandStrokePressed,
+    },
+    '@media (forced-colors: active)': {
+      '::after': {
+        backgroundColor: 'ButtonText',
+      },
+      ':hover::after': {
+        backgroundColor: 'ButtonText',
+      },
+      ':active::after': {
+        backgroundColor: 'ButtonText',
+      },
+    },
+  },
+});
+
+/**
+ * Styles for the icon slot.
+ */
+const useIconStyles = makeStyles({
+  base: {
+    gridColumnStart: 1,
+    gridRowStart: 1,
+    alignItems: 'center',
+    display: 'inline-flex',
+    justifyContent: 'center',
+    ...shorthands.overflow('hidden'),
+    [`& .${iconClassNames.filled}`]: {
+      display: 'none',
+    },
+    [`& .${iconClassNames.regular}`]: {
+      display: 'inline',
+    },
+  },
+  selected: {
+    [`& .${iconClassNames.filled}`]: {
+      display: 'inline',
+    },
+    [`& .${iconClassNames.regular}`]: {
+      display: 'none',
+    },
+  },
+});
+
+/**
+ * Styles for the content slot (children)
+ */
+const useContentStyles = makeStyles({
+  base: {
+    ...typographyStyles.body1,
+    ...shorthands.overflow('hidden'),
+    // content padding is the same for medium & small, horizontal & vertical
+    ...shorthands.padding(tokens.spacingVerticalNone, tokens.spacingHorizontalXXS),
+  },
+  selected: {
+    ...typographyStyles.body1Strong,
+  },
+  noIconBefore: {
+    gridColumnStart: 1,
+    gridRowStart: 1,
+  },
+  iconBefore: {
+    gridColumnStart: 2,
+    gridRowStart: 1,
+  },
+  placeholder: {
+    visibility: 'hidden',
+  },
+});
+
+/**
+ * Apply styling to the Tab slots based on the state
+ */
+export const useNavGroupStyles_unstable = (state: NavGroupState): NavGroupState => {
+  const rootStyles = useRootStyles();
+  const focusStyles = useFocusStyles();
+  const pendingIndicatorStyles = usePendingIndicatorStyles();
+  const activeIndicatorStyles = useActiveIndicatorStyles();
+  const iconStyles = useIconStyles();
+  const contentStyles = useContentStyles();
+
+  const { selected } = state;
+
+  state.root.className = mergeClasses(
+    navGroupClassNames.root,
+    rootStyles.base,
+    focusStyles.base,
+
+    // pending indicator (before pseudo element)
+    pendingIndicatorStyles.base,
```

Comment:
All styles that are applied unconditionally should ideally go into a single `makeResetStyles` call.

Reply:
I ended up blowing away most of the styles except the most basic. Think I got what you were aiming for.